test/AbstractFactory/ReflectionBasedAbstractFactoryTest.php

```diff
@@ -14,17 +14,19 @@
 use Laminas\ServiceManager\AbstractFactory\ReflectionBasedAbstractFactory;
 use Laminas\ServiceManager\Exception\ServiceNotFoundException;
 use PHPUnit\Framework\TestCase;
-use Prophecy\Prophecy\ObjectProphecy;
```

Comment:
The class is still in use. See the DocBlock in line 24.

Reply:
Ok, added this back in.